lib/savvy/index.js

```diff
@@ -10,14 +10,17 @@ mage.core.config.setTopLevelDefault('savvy', path.join(__dirname, 'config.yaml')
 var ROUTE_STR = '/savvy';
 var ROUTE_RE = new RegExp('^' + ROUTE_STR + '/');
 
-var SOCK_PATH;
 var SOCK_FILE = mage.core.config.get(['savvy', 'sockfile']);
-if (process.platform === 'win32') {
-	SOCK_PATH = path.join('\\\\.\\pipe', process.cwd(), SOCK_FILE);
-} else {
-	SOCK_PATH = path.join(process.cwd(), SOCK_FILE + '.sock');
+
+if (!path.isAbsolute(SOCK_FILE)) {
+	SOCK_FILE = path.resolve(process.cwd(), SOCK_FILE);
 }
 
+var SOCK_PATH = SOCK_FILE + '.sock';
```

Comment:
Ah right, you use `SOCK_FILE` for windows and not `SOCK_PATH`. I too don't mind either way either. Was merely referencing what you mentioned in your issue
https://github.com/mage/mage/issues/26#issue-227301655: `More a general case, but it looks a bit strange to enforce the extension on this one`